src/app/pages/din-oversikt-page/Info.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import intlHelper from '../../utils/intlUtils';
+import { FormattedMessage, useIntl } from 'react-intl';
+import SectionPanel from '../../components/section-panel/SectionPanel';
+import PillIkon from '../../svg/pilIkon';
+import './info.less';
+import bemUtils from '../../utils/bemUtils';
+// import ExpandableInfo from '../../components/expandable-content/ExpandableInfo';
+
+const bem = bemUtils('info');
+
+const Info = () => {
+    const intl = useIntl();
+
+    return (
+        <SectionPanel
+            illustration={<PillIkon />}
+            illustrationPlacement="outside"
+            title={intlHelper(intl, 'page.dinOversikt.info')}>
+            <>
+                <span className={bem.block}>
```

Comment:
Ikke bruk span for å wrappe block-elementer. Enten dropp den, eller bruk div. Span kan kun inneholde andre inline elementer